lib/happo/server.rb

```diff
@@ -57,40 +60,46 @@ def h(text)
         {
           description: '<First> with "test"',
           viewport: 'small',
-          url: 'http://placehold.it/350x150',
+          previous: 'http://placehold.it/350x150',
+          diff: 'http://placehold.it/350x150',
+          current: 'http://placehold.it/350x150',
```

Comment:
It might be nice to use some different sizes for these, since that will likely happen in real life--particularly with height but also width sometimes too.